perf/smallbank_workload/src/smallbank_tranformer.rs

```diff
@@ -28,20 +29,66 @@ use sawtooth_sdk::signing;
 
 use playlist::bytes_to_hex_str;
 use playlist::make_addresses;
-use smallbank::SmallbankTransactionPayload;
+use smallbank::{SmallbankTransactionPayload, SmallbankTransactionPayload_PayloadType};
 
 /// Transforms SmallbankTransactionPayloads into Sawtooth Transactions.
 pub struct SBPayloadTransformer<'a> {
     signer: &'a signing::Signer<'a>,
+    signature_by_customer_id: HashMap<u32, String>,
 }
 
 impl<'a> SBPayloadTransformer<'a> {
     pub fn new(signer: &'a signing::Signer) -> Self {
-        SBPayloadTransformer { signer: signer }
+        SBPayloadTransformer {
+            signer: signer,
+            signature_by_customer_id: HashMap::new(),
```

Comment:
Can you give this a more meaningful name? Something that gives a hint about why you want to track this mapping?

Reply:
I think this map should actually be encapsulated by a separate object which exposes a simple interface for tracking dependencies for different customers.